go.mod

```diff
@@ -624,6 +625,7 @@ require (
 	github.com/DataDog/datadog-agent/comp/core/status/statusimpl v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/comp/core/tagger/types v0.56.2
 	github.com/DataDog/datadog-agent/comp/core/telemetry v0.56.0-rc.3
+	github.com/DataDog/datadog-agent/comp/core/workloadmeta/collectors/util/kubemetadata v0.56.2
```

Comment:
Should we be using `v0.56.0-rc.3` instead here? I noticed that you used `v0.56.2` in some previous PRs, but most of these modules are using `v0.56.0-rc.3` so it might be best to be consistent.

Reply:
ah that makes sense. Usually when I've been running `go mod tidy` it pulls down the latest version but I can keep it consistent